docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -49,6 +49,8 @@ get:
 
         - <code>apps.deployments</code> embed all deployment identifier, if the related app currently is in deployment.
 
+        - <code>apps.readiness</code> embed all readiness results, if the related app has readines checks defined.
```

Comment:
If there are no readiness checks, then .

The clarification led me to thing that if the app definition doesn't define any readiness checks, then nothing will be added to the result. But actually Marathon will add an empty array.

So I'd just remove the clarification and leave this as `embed all readiness check results`.